_students/edwin_acevedo.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Edwin Acevedo
+site: "http://www.edwinacevedo.com"
+image: "images/...jpg"
+colorimg: "images/colored-one.jpg"
+---
+Software developer via Nashville Software School. BarCamp Nashville 2015 chair. Husband, father, urban chicken farmer. Proud food opportunist.
```

Comment:
The bio needs to be after a bio: key inside the dashes. Sorry, the template was wrong, but the examples show what i mean